packages/concerto-core/lib/systemmodel.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const systemNamespace = 'org.accordproject.base';
+const systemFileName = '@' + systemNamespace + '.cto';
+const systemModel = `namespace org.accordproject.base
+abstract asset Asset {  }
+abstract participant Participant {  }
+abstract transaction Transaction identified by transactionId {
+  o String transactionId
+}
+abstract event Event identified by eventId {
+  o String eventId
+}`;
+
+module.exports = { systemNamespace, systemFileName, systemModel };
```

Comment:
Why do we need this file? I thought that this would be explicitly added by users and hosted in a models repo?

Reply:
Sorry 😦 I had forgotten to push my latest changes to the branch on the main repo. It should be good now. The "system" model (which is now empty as per our discussion) is:
```
namespace system
abstract asset Asset {  }
abstract transaction Transaction { }
abstract event Event { }
abstract participant Participant {  }
```

We still need to declare the names `Asset` `Transaction` `Event` and `Participant` so users can write `concept Cat { o Participant owner }` having that file is a convenient way to do that (it would be done purely in JavaScript).